src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const item of action.keysToRemove) {
+          if (Object.keys(state).includes(item)) {
+            delete state[item];
+          }
+        }
+        break;
+
+      case 'clear':
+        for (const item in state) {
```

Comment:
as you know, when using for-of loops, we iterate over arrays, thus it's okay to give such names as `item`, but when it comes to iterating over object properties, that's better to call the current value `key`